### PR TITLE
(maint) Remove EOL platforms

### PR DESCRIPTION
**Pull request by underscorgan**

### CODE REVIEW

File: templates/pbuilderrc.erb [hunk 1]
```diff
@@ -5,10 +5,11 @@ USEDEVSHM=no
 # Codenames for Debian suites according to their alias. Update these when
 # needed.
 UNSTABLE_CODENAME="sid"
-TESTING_CODENAME="stretch"
-STABLE_CODENAME="jessie"
+TESTING_CODENAME="buster"
```
Comment (mwaggett) [inline templates/pbuilderrc.erb]: Where does "buster" come from?

File: templates/pbuilderrc.erb [hunk 2]
```diff
@@ -5,10 +5,11 @@ USEDEVSHM=no
...
+STABLE_CODENAME="stretch"
 STABLE_BACKPORTS_SUITE="$STABLE_CODENAME-backports"
-OLD_STABLE_CODENAME="wheezy"
+OLD_STABLE_CODENAME="jessie"
+OLD_OLD_STABLE_CODENAME="wheezy"
```
Comment (mwaggett) [inline templates/pbuilderrc.erb]: lol

File: templates/pbuilderrc.erb (hunk 1 above)
Comment (underscorgan) [reply templates/pbuilderrc.erb]: https://www.debian.org/releases/testing/

File: templates/pbuilderrc.erb (hunk 2 above)
Comment (mwaggett) [reply templates/pbuilderrc.erb]: `OLD_STABLE_CODENAME_NEW`

File: templates/pbuilderrc.erb [hunk 3]
```diff
@@ -51,7 +52,7 @@ UBUNTU_MIRROR="us.archive.ubuntu.com"
 <% end %>
 
 # List of old debian/ubuntu suites that require the traditional apt mirror
-OLD_SUITES=("lucid" "precise" "squeeze" "stable" "testing" "trusty" "wheezy")
+OLD_SUITES=("wheezy")
```
Comment (mwaggett) [inline templates/pbuilderrc.erb]: why no more "trusty"?

Comment (underscorgan) [reply templates/pbuilderrc.erb]: re-adding.

Comment (underscorgan) [reply templates/pbuilderrc.erb]: updated

**Review by mwaggett — APPROVED** [review]

File: templates/pbuilderrc.erb [hunk 4]
```diff
@@ -51,7 +52,7 @@ UBUNTU_MIRROR="us.archive.ubuntu.com"
 <% end %>
 
 # List of old debian/ubuntu suites that require the traditional apt mirror
-OLD_SUITES=("lucid" "precise" "squeeze" "stable" "testing" "trusty" "wheezy")
+OLD_SUITES=("wheezy" "trusty")
```
Comment (mwaggett) [inline templates/pbuilderrc.erb]: it's only bothering me a liiiiittle that these are in reverse order 😄

**Pull request merged**